Cargo.toml

```diff
@@ -18,3 +18,12 @@ bitcoin = "0.11"
 rust-crypto = "0.2"
 rand = "0.4"
 secp256k1 = "0.8"
+bech32 = {git = "https://github.com/sgeisler/rust-bech32"} # https://github.com/clarkmoody/rust-bech32/pull/4
+chrono = "0.4"
+regex = "0.2"
+bit-vec = "0.4.4"
+nom = "3.2.1"
+error-chain = "0.11.0"
```

Comment:
Hmm, do you really need all these deps? Seems like a lot just to save a few lines on parsing code. See-also: dep guidelines in README.

Reply:
>  dep guidelines in README.

Thank you for pointing that out, I probably skipped right ahead to the todo's. Getting rid of nom and bit-vec will be easy (nom looked nice at the beginning, but isn't that well documented etc. and not really of much help, I just stuck with it because it was challenging and fun). I can probably live without error-chain/failure too. Loosing regex could be a bit annoying.